tests/integration_tests/superset_test_config.py

```diff
@@ -133,11 +133,10 @@ def GET_FEATURE_FLAGS_FUNC(ff):
 
 
 class CeleryConfig:
-    BROKER_URL = f"redis://{REDIS_HOST}:{REDIS_PORT}/{REDIS_CELERY_DB}"
-    CELERY_IMPORTS = ("superset.sql_lab",)
-    CELERY_RESULT_BACKEND = f"redis://{REDIS_HOST}:{REDIS_PORT}/{REDIS_RESULTS_DB}"
-    CELERY_ANNOTATIONS = {"sql_lab.add": {"rate_limit": "10/s"}}
-    CONCURRENCY = 1
+    broker_url = f"redis://{REDIS_HOST}:{REDIS_PORT}/{REDIS_CELERY_DB}"
```

Comment:
These config settings were [renamed](https://docs.celeryq.dev/en/stable/userguide/configuration.html#new-lowercase-settings) and will  be removed in Celery 6.0.